<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Status_Screen</class>
 <widget class="QWidget" name="HNS_Status_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>508</width>
     <height>248</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:189px;
	padding-right:189px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>BRIGHTNESS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Light Sensor Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Message Board Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_BrightStatus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_BrightMode">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_LightInput">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1808</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_Brightness">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LBL_Status_Title">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>DEVICE STATUS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Status_Back">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="GB_Radar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>548</width>
     <height>248</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:238px;
	padding-right:238px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}

QGroupBox:disabled{
	background-color:#CCCAC9;
}</string>
   </property>
   <property name="title">
    <string>RADAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_RadarStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel:disabled{
	color:#000000;
}</string>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_LastSpeed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>27 MPH</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_SpeedLog1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8/7/20 | 15:00 | 45 MPH</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_SpeedLog2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8/7/20 | 18:00 | 85 MPH</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_SpeedLog3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8/7/20 | 19:00 | 20 MPH</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GRP_Communications">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>330</width>
     <height>248</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:70px;
	padding-right:69px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>COMMUNICATIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_68">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_69">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_70">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last GPS SIgnal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last Override</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_IPAddr">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.13.100</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_Port">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_LastGPS">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6/6/2020 23:59</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_LastOverride">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6/4/2020 23:59</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>330</width>
     <height>248</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:127px;
	padding-right:127px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>POWER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Battery Voltage (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Battery Current (A)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_SolarCurrentTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solar Current (A)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_BattVoltage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>11.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_BattCurrent">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_SolarCurrent">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_SolarVoltageTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solar Voltage (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_SolarVoltage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>330</width>
     <height>248</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	background-color:#FFFFFF;
	border-style:outset;
	border-radius:12px;
	border-radius:12px;
	border-color:#707070;
	border-width:2px
}

QGroupBox::title{
	background-color:#656565;
	color:#FFFFFF;
	padding-left:122px;
	padding-right:122px;
	padding-bottom:5px;
	subcontrol-position: top left;
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}</string>
   </property>
   <property name="title">
    <string>SYSTEM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Box Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_OutsideTempTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Battery Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_BoxTemp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>102.7°F</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Status_OutsideTemp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>68.1°F</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
